README.md

```diff
@@ -93,7 +93,7 @@ npm install -g benchpress
 
 Run ngTasty benchmark
 ```
-protractor benchmarks/protractor.conf.js --specs benchmarks/dist/benchmark.spec.js
+npm run protractor
 ```
```

Comment:
There are many benchmark, and this case I want just run only the `dist`.
Your solution will make run all of them, and this is not the case.